techniques/system/common/1.0/promises.st

```diff
@@ -549,6 +559,14 @@ bundle agent check_log_system
 
     aix.syslogd_repaired::
       "/usr/bin/refresh -s syslogd";
+      
+    rsyslogd.check_rsyslog_version_present::
+      # 5.6.4 is the first version mentionning RepeatedMsgReduction in http://www.rsyslog.com/change-log/
+      # The option is probably older though.
+      "${g.rudder_tools}/check-rsyslog-version 5.6.4"
```

Comment:
hum, we have a conflict there. Classes defined by module are global, so if we hve a version 5.6.5, we'll have both rsyslog_version_ok and rsyslog_version_not_ok defined